etc/kayobe/stackhpc.yml

```diff
@@ -199,7 +199,7 @@ stackhpc_repo_epel_9_version: "{{ stackhpc_repo_distribution }}"
 
 # ELRepo 9
 stackhpc_repo_elrepo_9_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/elrepo/elrepo/el9/x86_64/{{ stackhpc_repo_elrepo_9_version }}"
-stackhpc_repo_elrepo_9_version: "{{ stackhpc_repo_distribution }}"
+stackhpc_repo_elrepo_9_version: "20230907T075311"
```

Comment:
I don't think this is the correct place to override this variable. I can't remember it exactly now but can find it for you if you need

Reply:
This change is not related to 9.3 upgrade, sorry. Please see my comment below.